src/components/OrderForm/index.tsx

```diff
@@ -181,10 +158,25 @@ export class OrderForm extends React.PureComponent<OrderFormProps, OrderFormStat
         const availablePrecision = type === 'buy' ? currentMarketBidPrecision : currentMarketAskPrecision;
         const availableCurrency = type === 'buy' ? from : to;
 
+        const priceErrorClass = classnames('error-message', {
+            'error-message--visible': priceFocused && !isPriceValid.valid,
+        });
+
+        const priceText = this.props.translate('page.body.trade.header.newOrder.content.price');
+        const amountText = this.props.translate('page.body.trade.header.newOrder.content.amount');
+
+        const submitButtonText = type === 'buy' ? (
+            translate('page.body.trade.header.newOrder.content.tabs.buy')
```

Comment:
What about this? In case it is always buy or sell

`const submitButtonText = translate(`page.body.trade.header.newOrder.content.tabs.${type}`);